makefile

```diff
@@ -25,7 +25,7 @@ help:
 venv:
 	@echo "Start creating virtual environment"
 	$(PYTHON) -m venv $(VENV_PATH)
-	# pip install pip==23.0.1
+	pip install --upgrade pip
```

Comment:
Do we need this one after fixes in `tutorials/pipeline/6_custom_messenger_interface.py`